Makefile

```diff
@@ -78,6 +78,13 @@ DESTTARGET := $(abspath $(DESTBINDIR)/$(TARGET))
 DESTCONFDIR := $(DESTDIR)/$(SYSCONFDIR)/$(CCDIR)
 DESTCONFIG := $(abspath $(DESTCONFDIR)/$(CONFIG_FILE))
 
+PAUSEDIR := pause
+PAUSEBIN := pause
+PAUSEDESTDIR := $(abspath $(DESTDIR)/var/lib/clear-containers/runtime/bundles/pause_bundle/bin/$(PAUSEBIN))
```

Comment:
Please can you re-use the existing variables (`PAUSEROOTPATH`, and `PAUSEBINRELPATH`) rather than hard-coding this path..